portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java

```diff
@@ -50,8 +59,23 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       return;
     }
 
-    // Retrieve Comments from Datastore
-    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
+    // Receive input on which blog we are retrieving comments from
+    int maxNumBlogs = 5;
```

Comment:
Since this constant is shared between classes, can you create a single class to hold constants like these, and reference that class in both your List, Delete, and Add code?